ajax/libs/algebra.js/0.2.5/algebra.js

```diff
@@ -0,0 +1,37 @@
+var Fraction = require('./src/fractions');
+var Expression = require('./src/expressions').Expression;
+var Equation = require('./src/equations');
+var Parser = require('./src/parser');
```

Comment:
@pvnr0082t  @dannyxx001  Does this work in browsers? Especially you didn't commit a `src` folder.